src/components/LandingFilter.jsx

```diff
@@ -0,0 +1,152 @@
+/* eslint-env jquery */
+import React, { useState, useEffect } from 'react';
+import { FiArrowRightCircle } from 'react-icons/fi';
+import {
+  Dropdown, Grid, Form
+} from 'semantic-ui-react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import theme from '../config/theme';
+
+const GoButton = styled(Link)`
+  font-size: 50px;
+  font-weight: bold;
+  color: ${theme.colours.maroon};
+  :hover {
+    color: ${theme.colours.maroon};
+    opacity: 0.8;
+  }
+`;
+
+const regionOptions = [
```

Comment:
can we extract these constant values into the config folder? like create a constant.js file with these constant values.